src/Datadog.Trace.Ci.Shared/CIEnvironmentValues.cs

```diff
@@ -66,6 +81,21 @@ public static void DecorateSpan(Span span)
             span.SetTagIfNotNullOrEmpty(CommonTags.GitCommit, Commit);
             span.SetTagIfNotNullOrEmpty(CommonTags.GitBranch, Branch);
             span.SetTagIfNotNullOrEmpty(CommonTags.GitTag, Tag);
+            span.SetTagIfNotNullOrEmpty(CommonTags.GitCommitAuthorName, AuthorName);
+            span.SetTagIfNotNullOrEmpty(CommonTags.GitCommitAuthorEmail, AuthorEmail);
+            if (AuthorDate != DateTimeOffset.MinValue)
```

Comment:
Should we use a nullable `DateTimeOffset` for `AuthorDate` and `CommitterDate`?

Reply:
You're right, done in b69d87d